
This is an attempt to get a biarch compiler without full multilib stuff.

  - Rene Rebe <rene@exactcode.de>

--- ./sysdeps/unix/sysv/linux/configure.in.orig	2004-07-06 06:11:40.000000000 +0200
+++ ./sysdeps/unix/sysv/linux/configure.in	2005-01-17 21:22:22.740704976 +0100
@@ -161,6 +161,9 @@
 /usr | /usr/)
   # 64-bit libraries on bi-arch platforms go in /lib64 instead of /lib
   case $machine in
+  *)
+    libc_cv_slibdir="/lib"
+    ;;
   sparc/sparc64 | x86_64 | powerpc/powerpc64 | s390/s390-64 | \
   mips/mips64/n64/* )
     libc_cv_slibdir="/lib64"
